tests/testthat/_snaps/table_lbt03.md

```diff
@@ -7,10 +7,10 @@
                         (N=86)              (N=84)                (N=84)       
       —————————————————————————————————————————————————————————————————————————
       Baseline                                                                 
-        n                 86                  84                    82         
-        Mean (SD)    17.57 (9.22)       19.20 (10.05)          17.96 (8.72)    
-        Median          15.00               16.00                  17.00       
-        Min - Max    7.00 - 69.00        6.00 - 64.00          5.00 - 70.00    
```

Comment:
I would like to follow up on this, joining thread

Reply:
this happened before. 


https://github.com/insightsengineering/scda.test/pull/161/files

Reply:
so I think that was a but introduced here back then

https://github.com/insightsengineering/tern/pull/1347/files

@Melkiades @edelarua

Reply:
I think the fix from Emily is that the na_rm = TRUE is not anymore sent down into the `summarize_change` function, but it seems weird to me anyhow that was calculating anything at all before and the statistical function did not change afaik